Update hosting-tiers-table.md (#226)
bcgov/platform-developer-docs: src/docs/platform-architecture-reference/hosting-tiers-table.md

| **Target up time (availability)** | Platform availability for the Silver hosting tier is 90% for single-node and 99.5% for multi-node deployments |
| [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | - Protected B: Stored at rest (in a database or file storage)<br> - Protected C: Processing only (must be stored elsewhere) |
| **Integrated HA, DR options**| TransportServerClaims|

> | **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|                  

Reviewer: ```suggestion
| **Openshift upgrade plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|                  
```

| **Target up time (availability)** | Platform availability for the Gold hosting tier is 99.95%, specifically for applications with multi-node deployments that have a geographic failover to the Gold DR cluster in the Calgary data centre, minimizing disruption. <br><br>Without DR, Gold maintains the same availability as Silver |
| [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | - Protected B: Stored at rest (in a database or file storage)<br> - Protected C: Processing only (must be stored elsewhere) |
| **Integrated HA, DR options**| - TransportServerClaims to connect between Gold and Gold DR<br> -Global Load Balancing between clusters for Active/Passive DR|

> | **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|

Reviewer: ```suggestion
| **Openshift upgrade plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|
```

| **Target up time (availability)** | Platform availability for the Emerald hosting tier is 90% for single-node application deployments and 99.5% for multi-node deployments |
| [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | Protected C - Storage and/or processing |
| **Integrated HA, DR options**| None|

> | **Underlying release plan** | - Extended Update Support (EUS) <br> - Only updates to even numbered releases of OpenShift|

Reviewer: ```suggestion
| **Openshift upgrade plan** | - Extended Update Support (EUS) <br> - Only upgrades to even numbered releases of OpenShift|
```

> | **Slate of supported operators**| - Horizontal Pod Autoscaler<br> - Vertical Pod Autoscaler<br> - Custom Metrics Autoscaler<br> - Pipelines: Tekton<br> - CrunchyDB<br> - ArgoCD<br> - Kyverno <br>- License required Red Hat integration<br> - Enterprise DB <br> - JBoss  EAP <br> - IMB MQ |

Reviewer: ```suggestion
| **Supported operators**| - Horizontal Pod Autoscaler<br> - Vertical Pod Autoscaler<br> - Custom Metrics Autoscaler<br> - Pipelines: Tekton<br> - CrunchyDB<br> - ArgoCD<br> - Kyverno <br>- License required Red Hat integration<br> - Enterprise DB <br> - JBoss  EAP <br> - IMB MQ |
```
